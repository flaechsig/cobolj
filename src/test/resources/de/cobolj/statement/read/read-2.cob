IDENTIFICATION DIVISION.
PROGRAM-ID. HELLO.

ENVIRONMENT DIVISION.
   INPUT-OUTPUT SECTION.
      FILE-CONTROL.
      SELECT STUDENT ASSIGN TO 'src/test/resources/de/cobolj/statement/read/input.txt'.

DATA DIVISION.
   FILE SECTION.
   FD STUDENT.
   01 STUDENT-FILE.
      05 STUDENT-ID PIC 9(5).
      05 NAME PIC A(25).
      
   WORKING-STORAGE SECTION.
   01 WS-STUDENT.
      05 STUDENT-ID PIC 9(5).
      05 NAME PIC A(25).

PROCEDURE DIVISION.
    OPEN INPUT STUDENT.
        READ STUDENT
            AT END DISPLAY 'End of Records'
            NOT AT END DISPLAY STUDENT-FILE
        END-READ
    CLOSE STUDENT.
STOP RUN.
