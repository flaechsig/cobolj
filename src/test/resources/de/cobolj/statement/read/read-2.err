Read-Test ohne INTO-Phrase