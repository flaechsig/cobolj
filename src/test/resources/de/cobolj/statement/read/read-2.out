00001Hans                     
00002Franz                    
End of Records
